src/odemis/acq/align/delphi.py

```diff
@@ -1027,6 +1047,10 @@ def _DoHoleDetection(future, detector, escan, sem_stage, ebeam_focus, manual=Fal
         return first_hole, second_hole, hole_focus
 
     finally:
+        try:
+            escan.spotSize.value = init_spot_size
+        except Exception:
+            pass  # no spot size available
```

Comment:
Just in case it fails before reading the spotSize e.g. trying to subscribe. May it's a bit over the top though.